app/views/searches/show.html.erb

```diff
@@ -7,6 +7,17 @@
   </div>
 <% end %>
 
+<% if @gems && @gems.size < 1 && @gems.respond_to?(:response) && (suggestions = @gems.response.response['suggest']) && suggestions.present? && suggestions['suggest_name'].present? && suggestions['suggest_name'][0]['options'].length > 0  %>
```

Comment:
Yes, this is definitely too silly :( I guess you lifted it from the [example application](https://github.com/elastic/elasticsearch-rails/blob/master/elasticsearch-rails/lib/rails/templates/index.html.dsl.erb#L50), where it doesn't look very well as well.

I guess this should be better supported by the gem itself, at least allowing `@gems.suggestions` as a safe method, to get rid of most of the defensivity. What do you think?